Added Studio docs (#2455)
iterative/dvc.org: content/docs/studio/get-started.md

experiments. You can also generate plots and compare experiments here.

## Access your first view

In your browser, open <https://viewer.iterative.ai>. Sign in with your Github,
Gitlab, or Bitbucket account.

![](/img/studio/login_home.png) _When you first login, an example view is
already created for you to explore, and you can add more views._

When you first login, you will find that there already exists a view connecting
to an example DVC project. Use this view to explore the features that DVC Studio
has to offer.

DVC Studio automatically identifies datasets, metrics and hyperparameters in
your ML experiments. Each view on the dashboard displays the metrics. In the
figure above, you can see that `avg_prec` and `roc_auc` metrics are displayed.

## Components of a view

You can dive deep into all the experiments committed to the repo. For this, open
the view by clicking on the view name (in this case, `example-get-started`).

A table will be generated as shown below. This includes metrics, hyperparameters
and information about the datasets. All the data is flattened and neatly
presented for you to evaluate and compare the experiments.

> ![](/img/studio/main.png)

Reviewer: It's pretty hard to read anything in that image. I also noticed it's used in the index too. It's probably OK in the index but here we would need a smaller example or to filter out many of the columns so it's readable to this proportion more or less:

![image](https://user-images.githubusercontent.com/1477535/117898537-c4af8400-b28a-11eb-961f-062c44aa82ce.png)

Probably no need to show **Plots** yet in this first figure.

Reviewer: > For context: how it looks now (https://dvc-org-studio-docs-vntpxfoqdz.herokuapp.com/doc/studio/get-started#components-of-a-view):

![image](https://user-images.githubusercontent.com/1477535/117898604-f1fc3200-b28a-11eb-81b6-7dc500e3d8be.png)

your ML projects. You will need access to a Github, Gitlab or Bitbucket account
which has access to the Git repositories you want to connect to from DVC Studio.
DVC Studio creates views from repositories when you connect to them.

## What is a view?

A _view_ is an interactive representation of the experiments run in your ML
project. DVC Studio automatically identifies datasets, metrics and
hyperparameters in your ML experiments. You can also add custom files with the
required data (more on this later). Using this data, DVC Studio creates a
summary view. This includes tables of all the metrics across all your
experiments. You can also generate plots and compare experiments here.

## Access your first view

In your browser, open <https://viewer.iterative.ai>. Sign in with your Github,
Gitlab, or Bitbucket account.

![](/img/studio/login_home.png) _When you first login, an example view is
already created for you to explore, and you can add more views._

When you first login, you will find that there already exists a view connecting
to an example DVC project. Use this view to explore the features that DVC Studio

> has to offer.

Reviewer: > When you first login, you will find that there already exists a view connecting to an example DVC project.

This did not happen for me. I just see:

![image](https://user-images.githubusercontent.com/1477535/117891601-cd995900-b27c-11eb-9f21-86c4024025e8.png)

Reviewer: > Also the paragraph is repetitive with the img caption. I'd change the caption to _DVC Studio Views page_

Author: > This did not happen for me.
A separate task is ongoing to add the example view by default.

> change the caption to just DVC Studio Views page 
sounds good. will change.

Reviewer: > A separate task is ongoing to add the example view by default.

Meaning a docs issue? Didn't find this in #2468 or #2469

Author: there's a demo view that loads by default for new users now.

Reviewer: Great, thanks for the follow-up!

> ## Components of a view
> 
> DVC Studio automatically identifies datasets, metrics and hyperparameters in
> your ML experiments. Each view on the dashboard displays the metrics. In the
> example view shown above, you can see that `avg_prec` and `roc_auc` metrics are
> displayed.

Reviewer: That p seems to belong to the precious section (committing this):
```suggestion
DVC Studio automatically identifies datasets, metrics and hyperparameters in
your ML experiments. Each view on the dashboard displays the metrics. In the
figure above, you can see that `avg_prec` and `roc_auc` metrics are displayed.

## Components of a view
```

> # Get started

Reviewer: ```suggestion
# Get Started with DVC Studio
```
So that search result snippets/ embeds are different than for https://dvc.org/doc/start

Here, we will walk you through a tutorial to use DVC Studio for collaboration on

> your ML projects. You will need access to Github, Gitlab or Bitbucket account
> which has the Git repositories that you want to connect to from DVC Studio. When
> you connect to the repositories, DVC Studio creates views from those
> repositories.

Reviewer 2: ```suggestion
your ML projects. You will need access to a Github, Gitlab or Bitbucket account
which has access to the Git repositories you want to connect to from DVC Studio.
DVC Studio creates views from repositories when you connect to them.
```

## What is a view?

> A view is an interative presentation of data about the experiments that you have
> run in your ML project. DVC Studio automatically identifies datasets, metrics
> and hyperparameters in your ML experiments. You can also add custom files with
> the required data (more on this later). Using this data, DVC Studio creates a
> summary view, and you can drill down the summary view to visualize all the data
> across all your experiments in a table. You can also generate plots and compare
> experiments using the views.

Reviewer 2: ```suggestion
A *view* is an interactive representation of the experiments run in your ML
project. DVC Studio automatically identifies datasets, metrics and
hyperparameters in your ML experiments. You can also add custom files with the
required data (more on this later). Using this data, DVC Studio creates a
summary view. This includes tables of all the metrics across all your
experiments. You can also generate plots and compare experiments here.
```

## Access your first view

> In your browser, open https://viewer.iterative.ai/ Sign in with your Github,

Reviewer 2: ```suggestion
In your browser, open <https://viewer.iterative.ai>. Sign in with your Github,
```

Gitlab, or Bitbucket account.

![](/img/studio/login_home.png) _When you first login, an example view is
already created for you to explore, and you can add more views._

When you first login, you will find that there already exists a view connecting
to a public Git repository for an example DVC project. Use this view to explore
the features that DVC Studio has to offer.

## Components of a view

DVC Studio automatically identifies datasets, metrics and hyperparameters in
your ML experiments. Each view on the dashboard displays the metrics. In the
example view shown above, you can see that `avg_prec` and `roc_auc` metrics are
displayed.

You can dive deep into all the experiments committed to the repo. For this, open

> the view by clicking on the view name (in this case, example-get-started).

Reviewer 2: ```suggestion
the view by clicking on the view name (in this case, `example-get-started`).
```

the datasets. All the data is flattened and neatly presented for your to
visually evaluate the experiments as well as compare multiple experiements with
one another.

![](/img/studio/main.png) _Replace this image with an image that has labels for
the individual sections._

This tabular display has the following components:

1. The branches on your Git repository.
2. All the commits in the branches. Each commit represents an experiment. Each
   commit is on a single row in the table.
3. Values of all the metrics, files and parameters in the given commits. The
   different types of values are presented in different columns in the table.
4. Following buttons for performing actions:
   - **Filters:** Filter commits
   - **Columns:** Select columns to display
   - **Show plots:** Show plots, by selecting one or more commits that have
     plots
   - **Compare:** Compare different experiements
   - **Run:** Run experiments, by selecting any one commit (Refer here for how
     to run experiments)
   - **Trends:** Generate trend charts to show how the metrics eveolved over
     time
   - **Delta mode:** Change the displayed values to be differentials over the

>      values in the first row

Author: Accepted suggestion with minor variations

> You can connect to additional repositories and add more views as needed. In the
> next section, we will show you how you can do this.

Reviewer 2: ```suggestion
You can connect to additional repositories and add more views as needed. You can
find out how to do this in the next section.
```
